docs/content/TextArea.mdx

```diff
@@ -0,0 +1,187 @@
+---
+componentId: textarea
+title: TextArea
+description: Use TextArea for multi-line text input form fields
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/TextArea.tsx
+storybook: /react/storybook?path=/story/forms-textarea--default
+---
+
+import {TextArea} from '@primer/react'
+
+```js
+import {TextArea} from '@primer/react'
+```
+
+## Examples
+
+<Note variant="warning">
+
+TextArea components **must always** be accompanied by a corresponding label to improve support for assistive
+technologies. Examples below are provided for conciseness and may not reflect accessibility best practices.
+
+</Note>
+
+### Basic
+
+```javascript live noinline
+const TextAreaExample = () => {
+  // Running in controlled mode (recommended)
+  const [value, setValue] = React.useState('')
+
+  const handleChange = event => {
+    setValue(event.target.value)
+  }
+
+  return <TextArea onChange={handleChange}>{value}</TextArea>
```

Comment:
2 notes on the default example:

1. We should have a placeholder
2. Should it be a simple uncontrolled component?  just `<TextArea placeholder="something something">`

Reply:
Should we encouraging using placeholders by putting them in our examples? @alliethu [wrote up some feedback](https://github.com/primer/figma-post-to-github/issues/3#issue-1052084644) previously suggesting we shouldn't use them.

On point 2, I think we should encourage controlled over uncontrolled, but both examples exist. I also think our examples could generally be a bit more copy/paste-able so I went with full implementation detail. 

Good point though, I'll add a ultra minimal example without state handling too 👍

Reply:
re: placeholder: I read @alliethu's feedback as "don't use placeholders as labels, use labels." I think we should still use placeholders for giving examples. cc @alliethu what do you think?

Reply:
@siddharthkp, I think it's context dependent. Lots of times placeholders used as examples, are unnecessary as the label is straightforward enough. For text areas, however, I can imagine a placeholder being helpful in guiding the user on what type of content may be expected. 

@rezrah this would be a great example of how shifting left could solve for these types of questions early on. 😄

Reply:
Thanks all. Updated the basic example with a placeholder.